src/app.vue

```diff
@@ -1,14 +1,19 @@
 <template>
-  <BaseLabel>Hello World</BaseLabel>
+  <div class="flex-col d-flex flex-column align-items-center">
```

Comment:
We should remove everything from app.vue, and just render a simple `Hello World`. If we need to see components, we should go to storybook